quintype-build.config.js

```diff
@@ -4,7 +4,8 @@ const loadableConfigObj = {
   loadableConfig: {
     entryFiles: {
       topbarCriticalCss: "./app/isomorphic/components/layouts/header",
-      navbarCriticalCss: "./app/isomorphic/components/layouts/header/nav-bar"
+      navbarCriticalCss: "./app/isomorphic/components/layouts/header/nav-bar",
+      footerCriticalCss: "./app/isomorphic/components/layouts/footer"
```

Comment:
Does it make sense to name each of the CriticalCss. Just name topbar, navbar and footer.

Reply:
done